node/blockAPI/metaBlock_test.go

```diff
@@ -0,0 +1,331 @@
+package blockAPI
+
+import (
+	"encoding/hex"
+	"encoding/json"
+	"math/big"
+	"testing"
+
+	"github.com/ElrondNetwork/elrond-go/core"
+	"github.com/ElrondNetwork/elrond-go/data/api"
+	"github.com/ElrondNetwork/elrond-go/data/block"
+	"github.com/ElrondNetwork/elrond-go/dataRetriever"
+	"github.com/ElrondNetwork/elrond-go/node/mock"
+	"github.com/ElrondNetwork/elrond-go/storage"
+	"github.com/ElrondNetwork/elrond-go/testscommon"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMetaAPIBlockProcessor_GetBlockByHash_InvalidHashShouldErr(t *testing.T) {
```

Comment:
Maybe remove the second _ as to comply with the rest of the project. Valid also for L55, L251

Reply:
removed